python/hail/expr/aggregators/__init__.py

```diff
@@ -0,0 +1,33 @@
+from .aggregators import collect, collect_as_set, count, count_where, counter, \
+    any, all, take, min, max, sum, array_sum, mean, stats, product, fraction, \
+    hardy_weinberg, explode, filter, inbreeding, call_stats, info_score, hist, \
+    linreg
+
+__all__ = [
+    'collect',
+    'collect_as_set',
+    'count',
+    'count_where',
+    'counter',
+    'any',
+    'all',
+    'take',
+    'min',
+    'max',
+    'sum',
+    'array_sum',
+    'mean',
+    'stats',
+    'product',
+    'fraction',
+    'hardy_weinberg',
+    'explode',
+    'filter',
+    'inbreeding',
+    'call_stats',
+    'info_score',
+    'hist',
+    'linreg'
+]
+
+del aggregators
```

Comment:
shouldn't need to do this. People usually leave submodules/files in the namespace